src/odemis/util/test/focus_test.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 24 January 2025
+
+@author: Éric Piel
+
+Copyright © 2025 Éric Piel, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+
+import unittest
+import numpy
+from odemis.util.focus import MeasureSpotsFocus
+
+
+class TestMeasureSpotsFocus(unittest.TestCase):
```

Comment:
Can you add running these test cases to the cloud hosted ci? https://github.com/delmic/odemis/blob/master/.github/workflows/unit_tests_cloud.yml